### PR TITLE
Revise Polkadot.js Apps Page to Include Talisman

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

This page revises the Connect to Moonbeam with Polkadot.js Apps page replacing info on importing accounts to Polkadot.js Apps to the browser (which is no longer supported) with info on creating and injecting accounts via Talisman.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [x] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tokens/connect/talisman.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+description: Follow this quick tutorial to learn how to use Moonbeam’s Ethereum-standard H160 addresses and send transactions with Polkadot.js Apps and Talisman.
+---
+
+# Interacting with Moonbeam Using Talisman
+
+![Banner image](/images/tokens/connect/talisman/talisman-banner.png)
+
+## Introduction {: #introduction } 
+
+As a Polkadot parachain, Moonbeam uses a [unified account structure](/learn/features/unified-accounts/) that allows you to interact with Substrate (Polkadot) functionality and Moonbeam's EVM, all from a single Ethereum-style address. This unified account structure means that you don't need to maintain both a substrate and an Ethereum account to interact with Moonbeam - instead, you can do it all with a single Ethereum private key.
```
Comment (eshaben) [inline tokens/connect/talisman.md:12]:
```suggestion
As a Polkadot parachain, Moonbeam uses a [unified account structure](/learn/features/unified-accounts/){target=_blank} that allows you to interact with Substrate (Polkadot) functionality and Moonbeam's EVM, all from a single Ethereum-style address. This unified account structure means that you don't need to maintain both a Substrate and an Ethereum account to interact with Moonbeam - instead, you can do it all with a single Ethereum private key.
```

File: tokens/connect/talisman.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+This guide will include all of the steps for setting up an account in Talisman and using it to interact with Moonbeam through Polkadot.js Apps. 
+
+--8<-- 'text/disclaimers/third-party-content-intro.md'
+
+## Setting up Talisman {: #setting-up-talisman }
+
+Talisman is a crypto-wallet that natively supports Substrate (Polkadot) and Ethereum accounts. The Talisman wallet browser extension is available on Google Chrome, Brave and Firefox, and a corresponding asset dashboard is accessible at [app.talisman.xyz](https://app.talisman.xyz/){target=_blank}
+
+First, download and install the [Talisman extension](https://www.talisman.xyz/){target=_blank}. Once the extension opens up, you'll be prompted to either create a new wallet or import an existing one. For the purposes of this demo, we'll create a new wallet. On the following screen you'll be prompted to create a password to secure the new wallet.
```
Comment (eshaben) [inline tokens/connect/talisman.md:24]:
```suggestion
First, download and install the [Talisman extension](https://www.talisman.xyz/){target=_blank}. Once the extension opens up, you'll be prompted to either create a new wallet or import an existing one. For the purposes of this demo, you'll create a new wallet. On the following screen you'll be prompted to create a password to secure the new wallet.  
```

avoid using we or let's, etc. (unless it's a page in the tutorials section 🙂)

File: tokens/connect/talisman.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+This guide will include all of the steps for setting up an account in Talisman and using it to interact with Moonbeam through Polkadot.js Apps. 
+
+--8<-- 'text/disclaimers/third-party-content-intro.md'
+
+## Setting up Talisman {: #setting-up-talisman }
+
+Talisman is a crypto-wallet that natively supports Substrate (Polkadot) and Ethereum accounts. The Talisman wallet browser extension is available on Google Chrome, Brave and Firefox, and a corresponding asset dashboard is accessible at [app.talisman.xyz](https://app.talisman.xyz/){target=_blank}
```
Comment (eshaben) [inline tokens/connect/talisman.md:22]: links to download each would be nice

File: tokens/connect/talisman.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+To back up your newly created wallet, take the following steps:
+
+1. Press **Backup Now**
+2. Enter the password to your Talisman wallet
+3. Press **View Recovery Phrase** and store it in a secure place
+
+![Talisman prompt to create new wallet or import existing one](/images/tokens/connect/talisman/talisman-2.png)
+
+## Connecting Talisman to Polkadot.js Apps {: #connecting-talisman-to-polkadot.js-apps }
```
Comment (eshaben) [inline tokens/connect/talisman.md:39]:
```suggestion
## Connecting Talisman to Polkadot.js Apps {: #connecting-talisman-to-polkadot.js-apps }

```

File: tokens/connect/talisman.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+![Add Account in Talisman](/images/tokens/connect/talisman/talisman-6.png)
+
+Although our new account has been successfully created, Polkadot.js Apps isn't aware of it yet. To connect the new account to Polkadot.js Apps, take the following steps from [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmoonbeam-alpha.api.onfinality.io%2Fpublic-ws#/accounts){target=_blank}: 
+
+1. Open the Talisman extension and Press the **Connected / Not-connected** button
+2. Ensure **Show Eth accounts** is checked 
+3. Click on the account you'd like to connect. The green dot next to the account will light up if it is selected
+
+![Connect Talisman Account to Polkadot.js Apps](/images/tokens/connect/talisman/talisman-7.png)
```
Comment (eshaben) [inline tokens/connect/talisman.md:79]: nice job with this image ❤️

**Review by eshaben — CHANGES_REQUESTED** [review]
Awesomeee!! Looks good! Just a few comments 🙂

File: tokens/connect/polkadotjs.md [hunk 6]
```diff
@@ -27,9 +30,17 @@ After switching, the Polkadot.js site will not only connect to Moonbase Alpha, b
...
 ## Creating or Importing an H160 Account {: #creating-or-importing-an-h160-account } 
 
-In this section, you'll learn how you can create a new account, or import an already existing MetaMask account to Polkadot.js Apps.
+In this section, you'll learn how you can create a new account, or import an already existing MetaMask account to Polkadot.js Apps. First, there is one prerequisite step. As part of the process of phasing out support for accounts stored locally in the browser's cache, you'll need to enable support for local storage of accounts in the **Settings** tab. To do so, take the following steps:
+
+1. Navigate to the **Settings** Tab
+2. Select **Allow in-browser local account storage** under the **in-browser account storage** heading
+3. Press **Save**
+
+![Allow local in-browser account storage](/images/tokens/connect/polkadotjs/polkadotjs-9.png)
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply tokens/connect/polkadotjs.md]: Done: https://github.com/PureStake/moonbeam-docs-cn/pull/213

**Review by eshaben — APPROVED** [review]
LGTM! 🚀

**Pull request merged**